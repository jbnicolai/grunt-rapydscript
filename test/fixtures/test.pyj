def factorial(n):
    if n == 0:
        return 1
    return n * factorial(n-1)

factorial = def(n):
    if n == 0:
        return 1
    return n * factorial(n-1)
