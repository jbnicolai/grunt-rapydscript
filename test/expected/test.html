<!-- This page has been auto-generated using RapydML (http://www.pyjeon.com) -->
<html>
    <head>
        <title>
            Welcome to my Web Page
        </title>
    </head>
    <body>
        <div id="title">
            <img src="banner.png" alt="My Banner" />
        </div>
        <div id="content">
            I haven't yet put anything on this page
        </div>
        <div id="copyright">
            Copyright 2012 by Me
        </div>
    </body>
</html>
